.github/workflows/release.yml

```diff
@@ -182,8 +198,9 @@ jobs:
           fi
       - name: Publish to PyPI
         env:
-          POETRY_PYPI_TOKEN_PYPI: ${{ secrets.PYPI_API_TOKEN }}
-        run: make publish main=true
+          UV_PUBLISH_TOKEN: ${{ secrets.PYPI_API_TOKEN }}
+        run: |
+          make publish main=true
```

Comment:
Since `uv tree` includes the `v` in front of the version, will the pypi version include the `v` when running `uv publish`?

Reply:
I don't think so.